### PR TITLE
Canh/gas pricing

### PR DESCRIPTION
**Pull request by canhtrinh**

### CODE REVIEW

File: pages/dev/reference/pricing.mdx [hunk 1]
```diff
@@ -1,28 +1,59 @@
 # Axelar Pricing
 
-Transactions using Axelar often have a source chain, Axelar, and destination chains costs associated with them. To simplify this, Axelar has two general solutions to this problem. The first, for asset transfers, is to take a portion of the assets being transferred. The second is to use a chain-agnostic gas relaying service where gas is paid in the native source chain currency.
+Transactions using Axelar often have costs associated on each of their source chain, the Axelar network, and their destination chain. Whether for token transfers or General Message Passing, every cross-chain transaction is made up of 3 types of costs:
+1. initiating the transaction on the source chain,
+2. processing the transaction through the Axelar blockchain, and
+3. relaying to the gateway contract on the destination chain + executing the final transaction on the destination smart contract
```
Comment (milapsheth) [inline pages/dev/reference/pricing.mdx:6]:
```suggestion
1. Initiating the transaction on the source chain,
2. Processing the transaction through the Axelar blockchain
3. Relaying to the gateway contract on the destination chain
4. Executing the destination smart contract with the payload
```

File: pages/dev/reference/pricing.mdx [hunk 2]
```diff
@@ -1,28 +1,59 @@
 # Axelar Pricing
 
-Transactions using Axelar often have a source chain, Axelar, and destination chains costs associated with them. To simplify this, Axelar has two general solutions to this problem. The first, for asset transfers, is to take a portion of the assets being transferred. The second is to use a chain-agnostic gas relaying service where gas is paid in the native source chain currency.
+Transactions using Axelar often have costs associated on each of their source chain, the Axelar network, and their destination chain. Whether for token transfers or General Message Passing, every cross-chain transaction is made up of 3 types of costs:
```
Comment (milapsheth) [inline pages/dev/reference/pricing.mdx:3]:
```suggestion
Transactions using Axelar often have costs associated on each of their source chain, the Axelar network, and their destination chain. Whether for token transfers or General Message Passing, every cross-chain transaction is made up of 4 types of costs:
```

File: pages/dev/axelarjs-sdk/axelar-query-api.mdx [hunk 3]
```diff
@@ -24,23 +24,54 @@ const sdk = new AxelarQueryAPI({
 
 This is a useful query for GMP transactions, when invoking `callContract` or `callContractWithToken` to get an estimate of the appropriate gas payment to be made to the gas receiver on the source chain.
 
+IMPORTANT NOTES:
+
+* You must ensure the correct gasLimit is used for your gas estimation. The (optional) gasLimit parameter can be used here to override the defaukt 700K gas limit that is used.
```
Comment (StephenFluin) [inline pages/dev/axelarjs-sdk/axelar-query-api.mdx:29]: typo "defaukt" -> "default"

Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/axelar-query-api.mdx:29]: fixed

File: pages/dev/axelarjs-sdk/axelar-query-api.mdx [hunk 4]
```diff
@@ -24,23 +24,54 @@ const sdk = new AxelarQueryAPI({
 
 This is a useful query for GMP transactions, when invoking `callContract` or `callContractWithToken` to get an estimate of the appropriate gas payment to be made to the gas receiver on the source chain.
 
+IMPORTANT NOTES:
+
+* You must ensure the correct gasLimit is used for your gas estimation. The (optional) gasLimit parameter can be used here to override the defaukt 700K gas limit that is used. 
+* The minGasPrice and gasMultipler parameters can be adjusted to enhance execution certainty of cross-chain transactions.
```
[the remark this answers is not in the document]
Comment (canhtrinh) [reply pages/dev/axelarjs-sdk/axelar-query-api.mdx:30]: updated

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**